substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfDebugInfo.java

```diff
@@ -70,54 +71,56 @@ public class DwarfDebugInfo extends DebugInfoBase {
      */
     @SuppressWarnings("unused") public static final int DW_ABBREV_CODE_null = 0;
     /* Level 0 DIEs. */
-    public static final int DW_ABBREV_CODE_class_unit = 1;
+    public static final int DW_ABBREV_CODE_builtin_unit = 1;
```

Comment:
should we turn this into an enum or use a control variable to initialize the constants? Seems rather annoying that we have to bump each of them individually.

Reply:
Yeah that would probably be a good idea. However, I would prefer to defer that to a separate patch. I think this one may need backporting.